lang/en.js

```diff
@@ -25,11 +25,51 @@ export default {
       "dcSubject": "Subject",
       "dcTitle": "Title",
       "dcType": "Type of object",
-      "dctermsCreated": "Creation date",
+      "dcTermsCreated": "Creation date",
       "dcTermsMedium": "Medium",
       "edmCountry": "Providing country",
       "edmDataProvider": "Providing institution",
-      "edmRights": "License of the media in this record (unless otherwise specified)"
+      "edmRights": "License of the media in this record (unless otherwise specified)",
+      "dcRights": "Rights",
+      "dcDate": "Date",
+      "dcTermsPublished": "Publication Date",
+      "dcTermsIssued": "Issue date",
+      "dcTermsTemporal": "Temporal",
+      "dcCoverage": "Place-Time",
+      "dcTermsTOC": "Description",
```

Comment:
This is not a valid field. dcTermsTableOfContents is, but that should have the label "Table of contents" and there are no records with data for it anyway, so perhaps just remove it from here.